io_scene_godot/converters/animation/serializer.py

```diff
@@ -88,6 +88,8 @@ def factory(cls, xform_matrix, rotation_mode='QUATERNION'):
 
         if rotation_mode == 'QUATERNION':
             xform_frame.rotation_euler = xform_matrix.to_euler()
+        elif rotation_mode == 'AXIS_ANGLE':
```

Comment:
Could move this change into another pull request? so I can quickly merge it, thanks.

Also you can merge this two branch into `if rotation_mode in ('QUATERNION', 'AXIS_ANGLE')`

Reply:
fixed